notification/twilio/twiml.go

```diff
@@ -205,5 +207,8 @@ func (t *twiMLResponse) sendResponse() {
 	}
 
 	t.w.Header().Set("Content-Type", "application/xml; charset=utf-8")
-	io.WriteString(t.w, buf.String())
+	if _, err := io.WriteString(t.w, buf.String()); err != nil {
+		http.Error(t.w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		panic(err)
```

Comment:
We should not try to write a second set of response headers, nor panic the application if twilio disconnects early (the error state this represents)

Reply:
sure thing.  I simply copied the err pattern from earlier in the same function.  I've now updated it to ignore the error.